### PR TITLE
test(michelson encoder unit tests)

### LINKED ISSUE #1721: Ensure all Michelson types have unit test to make sure their plain JSON representation can be handled and converted
Refer to https://github.com/ecadlabs/taqueria/issues/892 from Taqueria.

### PR DESCRIPTION
**Pull request by jchenche**
Ensure all michelson types have unit tests for their plain JSON representation.

fix #1721

### CODE REVIEW

File: packages/taquito-michelson-encoder/test/tokens/bool.spec.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { BoolToken } from '../../src/tokens/comparable/bool';
+
+describe('Bool token', () => {
+  let token: BoolToken;
+  beforeEach(() => {
+    token = new BoolToken({ prim: 'bool', args: [{ prim: 'int' }], annots: [] }, 0, null as any);
```
Comment (roxaneletourneau) [inline packages/taquito-michelson-encoder/test/tokens/bool.spec.ts:6]: This doesn't change the behavior of the test, but I think that the args should be empty

File: packages/taquito-michelson-encoder/test/tokens/bool.spec.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  });
+
+  describe('EncodeObject', () => {
+    it('Should encode bool properly', () => {
```
Comment (roxaneletourneau) [inline packages/taquito-michelson-encoder/test/tokens/bool.spec.ts:10]: Consider using distinct names for tests in the same describe block to describe how tests are different.

File: packages/taquito-michelson-encoder/test/tokens/signature.spec.ts [hunk 3]
```diff
@@ -1,45 +1,102 @@
...
-    let token: SignatureToken;
-    beforeEach(() => {
-        token = new SignatureToken({ prim: 'signature', args: [], annots: [] }, 0, null as any);
+  let token: SignatureToken;
+  beforeEach(() => {
+    token = new SignatureToken({ prim: 'signature', args: [], annots: [] }, 0, null as any);
+  });
+
+  describe('EncodeObject', () => {
+    it('Should encode address to string', () => {
```
Comment (roxaneletourneau) [inline packages/taquito-michelson-encoder/test/tokens/signature.spec.ts:10]: Signature rather than address

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]
Thanks Jimmy, just some minor comments about test naming

**Comment by jchenche** [comment]
Thanks @roxaneletourneau comments addressed 🎉

**Review by roxaneletourneau — APPROVED** [review]
Looks good to me !

**Pull request merged**